iree/turbine/kernel/wave/utils/run_utils.py

```diff
@@ -109,13 +109,18 @@ def push_tensor_to_arg_list(arg_tensor: torch.Tensor):
     for input in inputs:
         if isinstance(input, torch.Tensor):
             push_tensor_to_arg_list(input)
-        else:
+        elif not isinstance(input, float):
             raise ValueError(f"Unsupported input type: {type(input)}")
     for output in outputs:
         if isinstance(output, torch.Tensor):
             push_tensor_to_arg_list(output)
         else:
             raise ValueError(f"Unsupported output type: {type(output)}")
+    # we want scalars to be at the end during codegen/dispatch to iree
+    # to maintain the consistency.
+    for input in inputs:
+        if isinstance(input, float):
```

Comment:
We should make this a notImplementedError here and a comment on how push_float is currently not working on IREE side